src/libexpr/json-to-value.cc

```diff
@@ -116,6 +116,12 @@ class JSONSax : nlohmann::json_sax<json> {
         return handle_value<void(Value&, const char*)>(mkString, val.c_str());
     }
 
+    bool binary(binary_t &)
```

Comment:
An alternative to overriding nlohmann_json is to make it build with both old and new versions, e.g.
```
#if NLOHMANN_JSON_VERSION_MAJOR >= 3 && NLOHMANN_JSON_VERSION_MINOR >= 8
     bool binary(binary_t &) override
#endif
...
```

Reply:
I'm reasonably sure that this code will compile under 3.7.x as well, because the existence of an additional, unused function in the class won't cause a compile error.

Reply:
Strike that, of course it won't, `binary_t` doesn't exist.  I'll make this adjustment.  Thanks for the eyes.